### PR TITLE
All lesson 5

### PR DESCRIPTION
**Pull request by awulw**

### CODE REVIEW

**Comment by awulw** [comment]
@tomekDiakowTieto

File: students/adam_wulw/lesson_05_lists/age_calculator.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+for age in age_list:
+    if age > 18:
+        adults.append(age)
+    else:
+        childrens.append(age)
+
+adults_mean_age = 0
+childrens_mean_age = 0
+for adult in adults:
+    adults_mean_age = adults_mean_age + adult
```
Comment (tomekDiakowTieto) [inline students/adam_wulw/lesson_05_lists/age_calculator.py]: You can use:
adults_mean_age += adult

File: students/adam_wulw/lesson_05_lists/comma_code.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+spam = ['apples', 'bananas', 'tofu', 'cats']
+
+
+def coma_code(_list):
+    _str = ''
```
Comment (tomekDiakowTieto) [inline students/adam_wulw/lesson_05_lists/comma_code.py]: I like this underscore

File: students/adam_wulw/lesson_05_lists/comma_code.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    for item in _list:
+        _str = _str + ' ' + str(item)
```
Comment (tomekDiakowTieto) [inline students/adam_wulw/lesson_05_lists/comma_code.py]: You can use += operator

File: students/adam_wulw/lesson_05_lists/swap_min_max.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+input_str = input()
+input_list = input_str.split()
+
+index_of_max = 0
+index_of_min = 0
+
+for i in range(0, len(input_list)):
+    if int(input_list[i]) > int(input_list[index_of_max]):
```
Comment (tomekDiakowTieto) [inline students/adam_wulw/lesson_05_lists/swap_min_max.py]: You can remove int(...) in every line.
You can user input_list = [int(item) for i in input_list] and later use it

**Pull request merged**